lib/services/authentication.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:convert';
+import 'dart:async';
+import 'package:http/http.dart' as http;
+import 'package:nativemoon/env.dart';
+
+class Authentication {
+  final String token;
+  final bool valid;
+
+  Authentication({this.token, this.valid});
+
+  factory Authentication.fromJson(Map<String, dynamic> json) {
+    return Authentication(token: json['jwt'], valid: true);
+  }
+}
+
+Future<Authentication> fetchAuthToken(String email, String password) async {
+  
+  final response = await http.post(env.baseUrl + '/api/auth/sign_in?email=' + email + '&password=' + password);
```

Comment:
@apedrob @ZePedroResende @herulume what do you think? I've redone the environment side of the app.

Reply:
The env variables shouldn't be on the source code or at least version control. Normally we use `dotenv` in JS, the same exists in [dart](https://pub.dev/packages/dotenv). Maybe @herulume knows better about this

Reply:
Those are external libraries implemented by the community, I tried not to create dependencies. The only way I found was this one.

Reply:
But if it's OK to use them I can change the implementation.

Reply:
I think the thing that should be underlined here is that this file with urls and other env vars cant be tracked, just a sample of it.

Reply:
I'm not sure if it's really used in dart but if `dotenv` is available I think it fits our needs perfectly @apedrob. We shouldn't do that work manually, fully agree with you. 
I would add a `.env.sample` file and tracked it as @ZePedroResende said.

TL;DR for me:
- Remove the `env.dart`
- Add `dotenv`
- Add a `.env.sample` file

Just a small note: there are some places in the code that still need to be replaced.

Reply:
I ended up using ```shel flutter_dotenv  ``` not  ```shell dotenv ```, I found it more specific for flutter.